src/main/webapp/app/shared/markdown-editor/markdown-editor.component.html

```diff
@@ -27,42 +27,48 @@
                     </div>
                     <!--color picker command -->
                     <div class="btn-group col-xs-6">
-                        <div class="color-preview form-control btn btn-outline-secondary" (click)="openColorSelector($event)">
+                        <div class="color-preview btn btn-sm" [ngStyle]="{ borderColor: selectedColor }" (click)="openColorSelector($event)">
                             Color
                         </div>
                         <jhi-color-selector [tagColors]="markdownColors" (selectedColor)="onSelectedColor($event)"></jhi-color-selector>
                     </div>
+
+                    <!-- domain commands -->
+                    <div class="markdown-editor__commands-domain" *ngIf="domainCommands && domainCommands.length != 0">
+                        <ng-container *ngFor="let command of domainCommands | typeCheck: DomainTagCommand">
+                            <div *ngIf="command.displayCommandButton === true" class="btn btn-sm" (click)="command.execute()">
```

Comment:
Can this be simplified to `command.displayCommandButton`?

Reply:
Good catch! I didn't notice it, while rearranging the commands. Fixed with https://github.com/ls1intum/Artemis/pull/1325/commits/cb8b3bbfc0d8d43792ef8f1033ee0fbfcb125008